src/index.js

```diff
@@ -1,34 +1,51 @@
+import PropTypes from 'prop-types';
 import React from 'react';
 import 'react-app-polyfill/ie11'; // For IE 11 support
 import ReactDOM from 'react-dom';
 import { OverridableContext } from 'react-overridable';
-import { Link } from 'react-router-dom';
 import 'semantic-ui-less/semantic.less';
-import { Item } from 'semantic-ui-react';
 import { InvenioILSApp } from './lib';
+import 'semantic-ui-less/semantic.less';
+import { Divider, Table } from 'semantic-ui-react';
+import _merge from 'lodash/merge';
+import _keys from 'lodash/keys';
 
-const CustomHome = ({ ...props }) => {
-  return <>And this is custom home</>;
-};
+const CDSDocumentMetadataExtensions = ({ extensions }) => {
```

Comment:
I would suggest to remove the word `CDS` from here, and replace it with `Custom` or `Example` or similar.

Reply:
PR #58 includes all these improvements